File: DependencyInjection/ServiceLocatorDIAntipattern/_2_ServiceLocator/InMessages/NullMessage.cs
using ServiceLocatorDIAntipattern._2_ServiceLocator.Interfaces;
using ServiceLocatorDIAntipattern._2_ServiceLocator.Services;

namespace ServiceLocatorDIAntipattern._2_ServiceLocator.InMessages
{
  class NullMessage : AcmeMessage
  {
    public void AuthorizeUsing(IAuthorization authorizationRules)
    {
      
    }

    public void WriteTo(DataDestination dataDestination)
    {
       
    }
  }
}

File: Java/dependency-injection/service-locator/src/main/java/ServiceLocatorAntipattern/InMessages/NullMessage.java
package ServiceLocatorAntipattern.InMessages;

import ServiceLocatorAntipattern.Interfaces.AcmeMessage;
import ServiceLocatorAntipattern.Interfaces.DataDestination;
import ServiceLocatorAntipattern.Services.IAuthorization;

public class NullMessage implements AcmeMessage {
  public void authorizeUsing(IAuthorization authorizationRules) {

  }

  public void writeTo(DataDestination dataDestination) {

  }
}